lib/data_update_scripts/20200217131245_re_index_existing_articles_with_approved.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class ReIndexExistingArticlesWithApproved
+    def run
+      Article.find_each { |article| Article.trigger_index(article, false) }
```

Comment:
```suggestion
      Article.published.find_each { |article| Article.trigger_index(article, false) }
```

We don't need to loop through unpublished stuff eh?

Reply:
hmmm, yeah that would make sense.